src/main/native/cpp/view/Analyzer.cpp

```diff
@@ -421,6 +429,31 @@ void Analyzer::Display() {
 
       ImGui::SetCursorPosY(endY);
 
+      // Add EPR and Gearing for converting Feedback Gains
+      ImGui::Separator();
+      ImGui::Spacing();
+
+      ImGui::SetNextItemWidth(ImGui::GetFontSize() * 10);
+      if (ImGui::InputDouble("Gearing", &m_gearing, 0.0, 0.0, "%.1f") &&
+          m_gearing > 0) {
+        Calculate();
+      }
+      sysid::CreateTooltip(
+          "This is the gearing between your encoder and your shaft");
+
+      ImGui::SetNextItemWidth(ImGui::GetFontSize() * 10);
+      if (ImGui::InputInt("EPR", &m_epr, 0) && m_epr > 0) {
+        Calculate();
+      }
+      sysid::CreateTooltip("This is the edges per rotation for your encoder");
+
+      if (ImGui::Checkbox("Convert Gains", &m_calcGains)) {
```

Comment:
I think we should have this at the top and only make the EPR and Gearing options show up if it's selected.